src/frontend/src/pages/GanttPage/GanttChartSection.tsx

```diff
@@ -48,6 +45,20 @@ const GanttChartSection = ({
     setCurrentTask(undefined);
   };
 
+  const handleMouseMove = (e: MouseEvent) => {
+    setCursorPosition({ x: e.clientX, y: e.clientY });
+  };
+
+  useEffect(() => {
+    if (!isEditMode) {
+      window.addEventListener('mousemove', handleMouseMove);
+    }
+
+    return () => {
+      window.removeEventListener('mousemove', handleMouseMove);
+    };
+  }, [isEditMode]);
```

Comment:
the update is fast enough to feel smooth rn, IMO we should remove the useEffect and instead of trying to make it feel super smooth, only make it pop up when u first hover over a new taskbar.
Upon looking into this a little more, I think a good solution is gonna involve limiting where we call handleMouseOver/ handleOnMouseLeave or restricting when cursor location is actually getting updated within those